optaplanner-core/src/main/java/org/optaplanner/core/api/compatibility/lang/NonNull.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.compatibility.lang;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.annotation.Nonnull;
+import javax.annotation.meta.TypeQualifierNickname;
```

Comment:
These come from the artifact `com.google.code.findbugs:jsr305:jar:1.3.9`, but I'd rather get them from a more canonical jar...

Reply:
Especially as we will get rid of guava soon.

Reply:
We got rid of guava on master. We need to figure out what's the best neutral artifact to get this from.

Reply:
We need to get them from JSpecify (credit: Anton at JFokus)